CondFormats/BTauObjects/src/BTagEntry.cc

```diff
@@ -17,8 +22,8 @@ BTagEntry::Parameters::Parameters(
   float discr_max
 ):
   operatingPoint(op),
-  measurementType(measurement_type),
-  sysType(sys_type),
+  measurementType(makeLowerCase(measurement_type)),
+  sysType(makeLowerCase(sys_type)),
```

Comment:
Well, actually it is not a local copy... 
The string `data` is put in lowercase and then you return a reference to the same string.  So you have at the end the modified string and a reference to it. I was thinking of having one copy with this:

````
static std::string makeLowerCase( std::string & data) {
  std::string lowercaseString;
  // Allocate the destination space
  lowercaseString.resize(data.size());
  // Convert the source string to lower case
  // storing the result in destination string
  std::transform(data.begin(), data.end(), lowercaseString.begin(), ::tolower);
  return lowercaseString;
}
````

Anyhow, let me see the static analyser...